editor/plugins/cpu_particles_2d_editor_plugin.cpp

```diff
@@ -217,11 +217,16 @@ void CPUParticles2DEditorPlugin::_generate_emission_mask() {
 	}
 
 	{
+		Vector2 offset(0., 0.);
```

Comment:
The default value is already `Vector2(0, 0)`, so you can skip the assignment. Also, we generally don't use C++-style parentheses initializers and prefer to use `=` instead where we do need assignment.

```suggestion
		Vector2 offset;
```

Reply:
Same... thank you.